{"name": "test_already_stick", "status": "broken", "statusDetails": {"message": "PermissionError: [Errno 13] Permission denied: 'F:\\\\python\\\\zrapiautotestv1.0\\\\TestData\\\\example.xlsx'", "trace": "self = <TestCase.StockQuotation.OptionalStock.test_stick_optional_stock.TestStickOptionalStock object at 0x12A5B1D0>\n\n    @allure.story('置顶自选股_已置顶')\n    def test_already_stick(self):\n        # 写\n>       write_xlsx(\"自选股\", 61, 7, str(self._id))\n\ntest_stick_optional_stock.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\..\\Common\\tools\\write_xlsx.py:11: in write_xlsx\n    wb.save(BASE_DIR + r'\\TestData\\example.xlsx')\nC:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\xlwt\\Workbook.py:710: in save\n    doc.save(filename_or_stream, self.get_biff_data())\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <xlwt.CompoundDoc.XlsDoc object at 0x12A67850>\nfile_name_or_filelike_obj = 'F:\\\\python\\\\zrapiautotestv1.0\\\\TestData\\\\example.xlsx'\nstream = b'\\t\\x08\\x10\\x00\\x00\\x06\\x05\\x00\\xbb\\r\\xcc\\x07\\x00\\x00\\x00\\x00\\x06\\x00\\x00\\x00\\xe1\\x00\\x02\\x00\\xb0\\x04\\xc1\\x00\\x02\\x00...13\\x00\\x02\\x00\\x00\\x00>\\x02\\x12\\x00\\xb6\\x00\\x00\\x00\\x00\\x00@\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\n\\x00\\x00\\x00'\n\n    def save(self, file_name_or_filelike_obj, stream):\n        # 1. Align stream on 0x1000 boundary (and therefore on sector boundary)\n        padding = b'\\x00' * (0x1000 - (len(stream) % 0x1000))\n        self.book_stream_len = len(stream) + len(padding)\n    \n        self._build_directory()\n        self._build_sat()\n        self._build_header()\n    \n        f = file_name_or_filelike_obj\n        we_own_it = not hasattr(f, 'write')\n        if we_own_it:\n>           f = open(file_name_or_filelike_obj, 'w+b')\nE           PermissionError: [Errno 13] Permission denied: 'F:\\\\python\\\\zrapiautotestv1.0\\\\TestData\\\\example.xlsx'\n\nC:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python36-32\\lib\\site-packages\\xlwt\\CompoundDoc.py:262: PermissionError"}, "start": 1594175253315, "stop": 1594175253404, "uuid": "603f9fd2-94c3-442f-9e18-61785964612c", "historyId": "e7ab01f60574fe407ff0077be774cf46", "testCaseId": "99988fe09260666df268754313a5ba2c", "fullName": "TestCase.StockQuotation.OptionalStock.test_stick_optional_stock.TestStickOptionalStock#test_already_stick", "labels": [{"name": "story", "value": "置顶自选股_已置顶"}, {"name": "parentSuite", "value": "TestCase.StockQuotation.OptionalStock"}, {"name": "suite", "value": "test_stick_optional_stock"}, {"name": "subSuite", "value": "TestStickOptionalStock"}, {"name": "host", "value": "DESKTOP-7K7NA43"}, {"name": "thread", "value": "19548-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.StockQuotation.OptionalStock.test_stick_optional_stock"}]}